src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -14,55 +17,46 @@ import { PdfDataFetchResponse } from "../common";
 interface DriverOverviewData {
     driverName: string | null;
     date: Date;
-    tableData: DriverOverviewTableData[];
+    tableData: DriverOverviewTablesData;
     message: string;
 }
 
-const compareDriverOverviewTableData = (
-    first: DriverOverviewTableData,
-    second: DriverOverviewTableData
-): number => {
-    if (first.name < second.name) {
-        return -1;
-    }
-    if (first.name > second.name) {
-        return 1;
-    }
-    return 0;
-};
-
-type ParcelsForDelivery = (Schema["parcels"] & {
+type ParcelForDelivery = Schema["parcels"] & {
     client: Schema["clients"];
+    collection_centre: Schema["collection_centres"];
     labelCount: number;
-})[];
+};
 
 type ParcelsForDeliveryResponse =
     | {
-          data: ParcelsForDelivery;
+          data: ParcelForDelivery[];
           error: null;
       }
     | {
           data: null;
           error: { type: ParcelsForDeliveryErrorType; logId: string };
       };
 
-type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient";
+type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient" | "noCollectionCentre";
 
 const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDeliveryResponse> => {
     const { data, error } = await supabase
         .from("parcels")
-        .select("*, client:clients(*), events(event_data)")
+        .select("*, client:clients(*), events(event_data), collection_centre:collection_centres(*), collection_centres(name), clients(address_postcode)")
         .in("primary_key", parcelIds)
         .limit(1, { foreignTable: "clients" })
+        .limit(1, { foreignTable: "collection_centres" })
         .eq("events.new_parcel_status", "Shipping Labels Downloaded")
-        .order("timestamp", { foreignTable: "events", ascending: false });
+        .order("collection_centres(name)")
+        .order("clients(address_postcode)");
+
```

Comment:
If you're interested, found the solution for this here: https://github.com/supabase/supabase-js/issues/97